src/main/kotlin/no/nav/helsearbeidsgiver/bro/sykepenger/db/ForespoerselDao.kt

```diff
@@ -279,7 +279,7 @@ private fun List<Pair<UUID, ForespoerselDto>>.finnNyesteForespoerselPerVedtakspe
                         forespoerselId = eksponertForespoerselId,
                     )
                 }
-        }
+        }.sortedBy { it.opprettet }
```

Comment:
Snek med denne urelaterte endringen. Den har ingen funksjonell effekt i dag, men gjør endringen for å få liste-outputen fra Dao-klassen mer konsekvent.